src/Generators/DocBlockGenerator.php

```diff
@@ -97,15 +108,14 @@ public function getGeneratedDocBlock()
      */
     protected function mergeGeneratedTagsIntoDocBlock($existingDocBlock)
     {
-        $docBlock = new DocBlock($this->removeExistingSupportedTags($existingDocBlock));
+        $docBlock = $this->docBlockFactory->create(($existingDocBlock ?: "/**\n*/"));
```

Comment:
I'm not 100% sure, but `\n` could/should be written as `PHP_EOL`, just to be sure?

Reply:
Only thing with `PHP_EOL` is on windows that resolves to `\r\n`, which PSR-2 and PSR-12 doesn't allow. That said ideally it should detect the files major EOL I would think and use that. I'll see if PHPDocumentor has anything for this or if we can come up with something quick for the detection.

Reply:
Hmmm, fair call.

Problem however, would be that some IDE's screw this up, and go with `\r\n` instead...

I agree, use `"\n"` and document the caveat for Windows users